react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx

```diff
@@ -470,3 +474,60 @@ BreakpointColorMap.parameters = {
         },
     },
 };
+
+export const ColorMapRange: ComponentStory<typeof DeckGLMap> = (args) => {
```

Comment:
Map3DDelatinMeshLayer/ColorMapRange example is interactive, it will be nice if we can make this story interactive too.

Reply:
Can I do that in next pull request? The story is copied from a simmilar story in terrain folder so I will have to figure out how to fix these.